modules/phase_field/src/action/MultiAuxVariablesAction.C

```diff
@@ -32,31 +33,35 @@ MultiAuxVariablesAction::MultiAuxVariablesAction(InputParameters params) :
 {
 }
 
+/*
+The name of the variable is the variable name base followed by two digits.
+The first digit in the name is the dimension it applies to:
+  0 = x, 1 = y, 2 = z.
+The second digit in the name is the order parameter it applies to.
+*/
+
 void
 MultiAuxVariablesAction::act()
 {
   MooseEnum order = getParam<MooseEnum>("order");
   MooseEnum family = getParam<MooseEnum>("family");
-  std::vector<unsigned int> op_num = getParam<std::vector<unsigned int> >("op_num");
+  unsigned int op_num = getParam<unsigned int>("op_num");
   std::vector<std::string> var_name_base = getParam<std::vector<std::string> >("var_name_base");
```

Comment:
@jwpeterson took a few unused members out recently. So the only warnings I am seeing when compiling in `dbg` mode are in the tensor mechanics module.